source_gen/test/builder_test.dart

```diff
@@ -296,13 +296,13 @@ void main() {
           header: '',
         ),
         {
-          '$_pkgName|lib/a.dart': '''// @dart=2.9
+          '$_pkgName|lib/a.dart': '''// @dart=2.12
```

Comment:
This is only in generated test code.

Bumping to 2.12 to avoid the falso positive.